pandas/core/base.py

```diff
@@ -7,8 +7,10 @@
 from pandas.core import common as com
 import pandas.core.nanops as nanops
 import pandas.lib as lib
+from pandas.compat.func_defaults import TRANSPOSE_DEFAULTS
```

Comment:
Right, I think I get what you're saying in terms of abstracting away all of the validation machinery into one function call.